README.md

```diff
@@ -631,7 +631,10 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
 
 - `drop_debugger` (default: `true`) -- remove `debugger;` statements
 
-- `evaluate` (default: `true`) -- attempt to evaluate constant expressions
+- `evaluate` (default: `true`) -- Evaluate expression for shorter constant
+  representation. Pass `"eager"` to always replace function calls whenever
+  possible, or a positive integer to specify an upper bound for expansion in
+  number of characters.
```

Comment:
s/for expansion/for each individual evaluation/

Reply:
Done in https://github.com/mishoo/UglifyJS2/pull/3587/commits/63ca6cc81b5e1925b8f731ccc3a29cbc7594c302